13-tool-usage-cycle-tracking/README.md

```diff
@@ -290,8 +357,15 @@ With a 120v supply, this configuration monitors input power to the relay, and th
 | `heartbeat_mins`          | 5      | Send power monitoring events every 5 minutes. |
 | `alert_under_voltage`     | 100    | Send an alert when voltage is less than 100v. |
 | `alert_over_voltage`      | 135    | Send an alert when voltage is above 135v. |
-| `switch2`                 | true/false | Control power to the equipment to be on or off. |
-| `alert_power_activity_2`  | supply | Correlate the equipment power with the state of the relay. |
+| `input2`                  | true   | Sense load activity via input 2 (`D11`) |
+| `alert_power_activity_2`  | supply | Correlate the tool power with activity |
+| `power_activity_startup_secs_2` | 10 | allow 10 seconds for the tool to start up |
+| `power_activity_shutdown_secs_2` | 10 | allow 10 seconds for the tool to shut down |
+| `alert_off_vibration` | 10 | minimal vibration when the tool is not in use |
+| `alert_under_vibration` | 100 | vibration level with a margin below the lowest vibration sensed when the tool is active |
+| `alert_over_vibration` | 150 | vibration level with a margin above the highest vibration sensed when the tool is active |
+| `alert_vibration_activity_line` | 2 | Correlate sensed vibration with tool activity from input 2 |
```

Comment:
I’d capitalize the first letter of each description above and add periods to the end (for consistency with the existing descriptions).